ml/nemo_xgboost_optimization.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/python3
```

Comment:
Please describe this file briefly tomorrow.

Reply:
Done 😄